src/api/tickets/resetTicketStatistics.spec.ts

```diff
@@ -1,67 +1,85 @@
 import nock from 'nock';
 import { sdkApiError } from '../../utils';
-import { getTicketStatistics } from './getTicketStatistics';
+import { resetTicketStatistics } from './resetTicketStatistics';
+import { ZodError } from 'zod';
 
 const API_ENDPOINT = 'http://localhost:3001';
 const API_TOKEN = 'S3CR3T-T0K3N';
 
-describe('test getTicketStatistics', function () {
+describe('test resetTicketStatistics', function () {
   beforeEach(function () {
     nock.cleanAll();
   });
-  it('handles successful response', async function () {
-    nock(API_ENDPOINT).get(`/api/v3/tickets/statistics`).reply(200, {
-      neglectedValue: 'string',
-      redeemedValue: 'string',
-      rejectedValue: 'string',
-      unredeemedValue: 'string',
-      winningCount: 0
-    });
 
-    const response = await getTicketStatistics({
+  it('handles successful response (204)', async function () {
+    nock(API_ENDPOINT).delete(`/api/v3/tickets/statistics`).reply(204);
+
+    const response = await resetTicketStatistics({
       apiToken: API_TOKEN,
       apiEndpoint: API_ENDPOINT
     });
 
-    expect(response.winningCount).toEqual(0);
+    expect(response).toEqual(true);
+  });
+
+  it('throws an Error when server returns 500+ error', async function () {
+    nock(API_ENDPOINT).delete(`/api/v3/tickets/statistics`).reply(500, 'Server Error');
+
+    await expect(
+      resetTicketStatistics({ apiToken: API_TOKEN, apiEndpoint: API_ENDPOINT })
+    ).rejects.toThrow(Error);
   });
-  it('throws a custom error when hoprd api response is an 400 error', async function () {
-    nock(API_ENDPOINT).get(`/api/v3/tickets/statistics`).reply(400, {
+
+  it('throws a custom error when hoprd api response is a 400 error', async function () {
+    nock(API_ENDPOINT).delete(`/api/v3/tickets/statistics`).reply(400, {
```

Comment:
This function doesn't get peer ids as an input so it can't return INVALID_PEERID